inference-engine/thirdparty/clDNN/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -5353,7 +5353,99 @@ TEST_P(convolution_gpu_fs_byx_fsv32_crop, fs_byx_fsv32_crop)
                 }
 }
 
+TEST(convolution_f32_fw_gpu, convolution_int8_b_fs_yx_fsv4_to_bfyx) {
```

Comment:
Test coverage should be extended for these kernels.

Reply:
Fused_conv_eltwise kernel removed from this PR so no test needed for it

Reply:
Are there any test cases in fusings_gpu_tests with this kernel?